src/azure-cli/azure/cli/command_modules/util/custom.py

```diff
@@ -0,0 +1,31 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from __future__ import print_function
```

Comment:
We don't support Python 2.0+ anymore, right?
Do we still need `from __future__ import print_function`? [print-as-a-function](https://docs.python.org/3/whatsnew/2.6.html?highlight=print_function#pep-3105-print-as-a-function)

Reply:
Nice catch! I simply migrated the code from `src/azure-cli/azure/cli/command_modules/resource/custom.py`. Removed.